game-backend/src/game-engine/recordIntents.test.ts

```diff
@@ -1,39 +1,64 @@
+import StateBuilder from '../test-helper/StateBuilder';
 import recordIntents from './recordIntents';
+import { State } from '../../../darwin-types/State';
+import { UserExecutionContext } from '../../../darwin-types/UserContext';
+
+const UNIT_ID = 'unit1';
+let emptyState: State;
+let userExecutionContext: UserExecutionContext;
```

Comment:
instance variables of tests should be contained in the `describe` callback function.